frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/api/BrokerAddress.java

```diff
@@ -0,0 +1,17 @@
+package com.mesosphere.sdk.kafka.api;
+
+import com.mesosphere.sdk.api.types.EndpointProducer;
+
+/**
+ * Endpoint Resource.
+ */
+public class BrokerAddress implements EndpointProducer {
```

Comment:
Per below this can be removed in favor of the automatic VIP-based `broker` entry.